docs/UserGuide.md

```diff
@@ -108,43 +110,44 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
 
 Examples:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st property to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd property to be `Betsy Crower` and clears all existing tags.
+*  `edit 2 n/Blk 298 Toa Payoh Central t/` Edits the name of the 2nd property to be `Blk 298 Toa Payoh Central` and clears all existing tags.
 
```

Comment:
```md
*  `edit 2  ta/3rm td/4rm` Removes the `4rm` tag and adds the `3rm` tag to the currently available tag list of the 2nd property.
*  `edit 2 t/condo  ta/3rm td/4rm` Edits the 2nd property by clearing all existing tags to only `condo`, then removes the `4rm` tag (if existing) and adds the `3rm` tag to the current tags.
```
Note: The second pointer is more dubious and unconfirmed functionality. We may not allow mixing of `t/` with `ta/` or `td/`. The first pointer, however, should be safe.